operations/reservations.md

```diff
@@ -693,8 +695,10 @@ Updates information about the specified reservation. Note that if any of the fie
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
 | `Reason` | string | optional | Reason for updating the reservation. Required when updating the price of the reservation. |
-| `CheckOverbooking` | bool | optional | Whether reservation overbooking is checked. Default value is `true`.  |
-| `CheckRateApplicability ` | bool | optional | Whether the rate applicability check is checked. Default value is `true`.  |
+| `CheckOverbooking` | bool | optional | Whether reservation overbooking is checked. Default value is `true`. |
+| `CheckRateApplicability` | bool | optional | Whether the rate applicability is checked. If set to `false` it allows to set a restricted rate. Default value is `true`. |
+| `Reprice` | bool | optional | Whether the price should be updated to latest value for date/rate/category combination set in Mews. Default value is `true`. |
+| `ApplyCancellationFee` | bool | optional | Whether the cancellation fee should be applied on cancelled nights. Default value is `true`. |
```

Comment:
I saw just nights maybe I am overseeing something

Reply:
The naming is not great but it depends on extent set up on cancellation policy. and it can include product orders.
![image](https://user-images.githubusercontent.com/435787/103631558-691f6380-4f43-11eb-96e7-2ecc63bca992.png)

So I'd rephrase this to something like "Whether cancellation fees should be applied according to rate cancellation policies." Or something along those lines.

Also I see the phrase "Default value is `true`." That's inconsistent with the rest of the documentation, where we use "If not specified, ..."
![image](https://user-images.githubusercontent.com/435787/103631844-cc10fa80-4f43-11eb-98c3-a199cab0c0e2.png)